jsonrpc/eth_endpoint.go

```diff
@@ -91,27 +91,29 @@ func (e *Eth) ChainId() (interface{}, error) {
 	return argUintPtr(e.chainID), nil
 }
 
-func (e *Eth) getHeaderFromBlockNumberOrHash(bnh *BlockNumberOrHash) (*types.Header, error) {
-	var (
-		header *types.Header
-		err    error
-	)
+func (e *Eth) getHeaderFromBlockNumberOrHash(bnh BlockNumberOrHash) (*types.Header, error) {
+	// The filter is empty, use the latest block by default
+	if bnh.BlockNumber == nil && bnh.BlockHash == nil {
```

Comment:
Could we have one switch for all cases there?

Reply:
Let's address that on a followup PR.